exercises/simple-linked-list/simple-linked-list.js

```diff
@@ -0,0 +1,37 @@
+//
+// This is only a SKELETON file for the 'Simple Linked List' exercise. It's been provided as a
+// convenience to get you started writing code faster.
+//
+
+export class SimpleLinkedList {
+
+  constructor() {
+    throw new Error("Remove this statement and implement this function");
+  }
+
+  add(nextValue) {
+    throw new Error("Remove this statement and implement this function");
+  }
+
+  remove(removeValue) {
+    throw new Error("Remove this statement and implement this function");
+  }
+
+  length() {
+    throw new Error("Remove this statement and implement this function");
+  }
+
+  fromArray(numberArray) {
+    throw new Error("Remove this statement and implement this function");
+  }
+
+  toArray() {
+    throw new Error("Remove this statement and implement this function");
+  }
+
+  reverse() {
+    throw new Error("Remove this statement and implement this function");
+  }
+
```

Comment:
we also need a `head` function which will be a getter. Also `fromArray` & `toArray` are not required. Please refer [spec file](https://github.com/exercism/javascript/blob/master/exercises/simple-linked-list/simple-linked-list.spec.js) for required functions.

Reply:
- Added a new getter property called `head`.
- Removed the `fromArray` method and kept the `toArray` method as it is, as it is required as per the [spec file](https://github.com/exercism/javascript/blob/master/exercises/simple-linked-list/simple-linked-list.spec.js).